tests/__init__.py

```diff
@@ -0,0 +1 @@
+from .test_foo import *  # noqa
```

Comment:
tests? wie geth das denn? (warum sei mal nicht gefragt, exemplarisch als lernbeispiel kann das ja gut sein)

Reply:
https://beust.com/weblog/2006/08/03/untested-code-is-the-dark-matter-of-software/ \*hust\*